src/components/TimelineEntry.tsx

```diff
@@ -0,0 +1,160 @@
+import { PropsWithChildren } from 'react'
+
+import { TimelineEntryStatus, TimelinePosition } from '../lib/types'
+
+export interface TimelineEntryProps extends PropsWithChildren {
+  type: TimelineEntryStatus
+  step: string
+  date?: string
+  position: TimelinePosition
+  className?: string
+  background?: boolean
+}
+
+const topBorderStyle = (
+  type: TimelineEntryStatus,
+  position: TimelinePosition,
+) => {
+  const style =
+    type == 'future' ? 'border-dashed border-black' : 'border-accent-success'
+
+  return position == 'first' ? 'border-transparent' : style
+}
+
+const bottomBorderStyle = (
+  type: TimelineEntryStatus,
+  position: TimelinePosition,
+) => {
+  const style =
+    type == 'done' ? 'border-accent-success' : 'border-dashed border-black'
+
+  return position == 'last' ? 'border-transparent' : style
+}
+
+const svgStyles = {
+  done: '-translate-x-8',
+  current: '-translate-x-8',
+  future: '-translate-x-8',
+}
+
+const SVG = (type: TimelineEntryStatus, background: boolean | undefined) => {
```

Comment:
It would be a good idea to start using [Font-Awesome](https://docs.fontawesome.com/v5/web/use-with/react) icons instead of hardcoding svg in our code.

Reply:
Won't pull this in immediately, but I agree in principle we should have a more standardized system for these SVG's. But at least if it's something we made, we can't run afoul of any usage restrictions, and it does make the dependencies simpler. I'll add an investigation item to the backlog to look into what things would look like with an icon pack, and whether business would approve the change in look.